packages/aws-cdk-lib/aws-workspacesthinclient/.jsiirc.json

```diff
@@ -4,7 +4,7 @@
       "package": "software.amazon.awscdk.services.workspacesthinclient"
     },
     "dotnet": {
-      "package": "Amazon.CDK.AWS.WorkSpacesThinClient"
+      "namespace": "Amazon.CDK.AWS.WorkSpacesThinClient"
```

Comment:
@vlesierse the best fix for all of these is to actually fix the source that codegens new packages. I assume `package` and `namespace` are configs that do the same thing right?

The file gets generated here: https://github.com/aws/aws-cdk/blob/704f596539372364408b6bff83e21221b6bb0970/packages/aws-cdk-lib/scripts/submodules/index.ts#L54-L75

Reply:
Good idea to change this at the source, was missing that. So a package it the NuGet package name, which is correct, but because CDK v2 put everything in the AWS.CDK.Lib package, the module needs to be specified on the namespace.

Reply:
If you look at the other modules, it is done correctly.